Python/executor_cases.c.h

```diff
@@ -3936,9 +3928,7 @@
                 stack_pointer[-1] = value;
                 PyStackRef_CLOSE(tmp);
                 stack_pointer = _PyFrame_GetStackPointer(frame);
-                stack_pointer[-1] = b;
             }
-            stack_pointer[-1] = b;
```

Comment:
@markshannon, this change seems incorrect. `b` is never written to the stack if we take the first conditional (where `value` is `None` and `b` is `True`). I suspect there might be other places where reasoning about both arms of a conditional is broken in a similar way.

Reply:
https://github.com/python/cpython/pull/131773